### PR TITLE
BZ1979956_4.9: Terminology consistency - use of term "provisioner"

### PR DESCRIPTION
**Pull request by tmulquee**
Version:
  * PR applies only to a specific single version: 4.9
Issue: https://bugzilla.redhat.com/show_bug.cgi?id=1979956
Link to docs preview: Link to docs preview:
http://file.emea.redhat.com/tmulquee/BZ1979956_4.9/installing/installing_bare_metal_ipi/ipi-install-overview.html
Overview: more consistent terminology and improved flow description.

### CODE REVIEW

**Comment by stevsmit** [comment]
@tmulquee would you mind fixing the preview? This one isn't generating.

**Comment by tmulquee** [comment]
@stevsmit When I looked at the preview, it's there but for some reason doesn't load first time. If you hit Ctrl-R, it loads the second time. I'll regenerate it to see if that has any effect on the issue.

**Comment by tmulquee** [comment]
Preview working fine now.

File: installing/installing_bare_metal_ipi/ipi-install-overview.adoc [hunk 1]
```diff
@@ -4,19 +4,25 @@
...
+For the installation, the key elements in the previous diagram are:
+
+- **Provisioner**: A physical machine that runs the installation program and hosts the bootstrap VM that deploys the controller of a new {product-title} cluster.
+- **Bootstrap VM**: A virtual machine that assists in the process of deploying an {product-title} cluster.
+- **Network bridges**: The bootstrap VM connects to the bare-metal network and to the provisioning network, if present, via network bridges, `eno1` and `eno2`.
+
+In phase 2 of the deployment, the provisioner destroys the bootstrap VM automatically and moves the virtual IP addresses (VIPs) to the appropriate nodes. The API VIP moves to the control plane nodes and the Ingress VIP moves to the worker nodes.
 
-When the installation of {product-title} is complete and fully operational, the installation program destroys the bootstrap VM automatically and moves the virtual IP addresses (VIPs) to the appropriate nodes accordingly. The API VIP moves to the control plane nodes and the Ingress VIP moves to the worker nodes.
+The following diagram illustrates phase 2 of deployment.
```
Comment (stevsmit) [inline installing/installing_bare_metal_ipi/ipi-install-overview.adoc:19]: s/ . / :

File: installing/installing_ibm_cloud/install-ibm-cloud-prerequisites.adoc [hunk 2]
```diff
@@ -10,15 +10,15 @@ You can use installer-provisioned installation to install {product-title} on IBM
 
 [IMPORTANT]
 ====
-Red Hat supports IPMI and PXE on the `provisioning` network only. Red Hat has not tested Red Fish, virtual media, or other complementary technologies such as Secure Boot on IBM Cloud deployments. The `provisioning` network is required.
+Red Hat supports IPMI and PXE on the provisioning network only. Red Hat has not tested Red Fish, virtual media, or other complementary technologies such as Secure Boot on IBM Cloud deployments. A provisioning network is required.
 ====
 
 Installer-provisioned installation of {product-title} requires:
```
Comment (stevsmit) [inline installing/installing_ibm_cloud/install-ibm-cloud-prerequisites.adoc:16]: requires the following:

File: modules/ipi-install-troubleshooting-bootstrap-vm.adoc [hunk 3]
```diff
@@ -94,13 +94,13 @@ $ ssh core@172.22.0.2
 
 If you are not successful logging in to the bootstrap VM, you have likely encountered one of the following scenarios:
 
-* You cannot reach the `172.22.0.0/24` network. Verify network connectivity on the provisioner host specifically around the `provisioning` network bridge. This will not be the issue if you are not using the `provisioning` network.
-
+* You cannot reach the `172.22.0.0/24` network. Verify the network connectivity between the provisioner and the `provisioning` network bridge. This issue may occur if you are using a `provisioning` network.
```
Comment (stevsmit) [inline modules/ipi-install-troubleshooting-bootstrap-vm.adoc:97]: s/ may / might (IBMSG recommends against using "may"

File: installing/installing_bare_metal_ipi/ipi-install-overview.adoc [hunk 4]
```diff
@@ -4,19 +4,25 @@
 include::_attributes/common-attributes.adoc[]
 :context: ipi-install
 
-Installer-provisioned installation provides support for installing {product-title} on bare metal nodes. This guide provides a methodology to achieving a successful installation.
-
-During installer-provisioned installation on bare metal, the installation program on the bare metal node labeled as `provisioner` creates a bootstrap virtual machine (VM). The role of the bootstrap VM is to assist in the process of deploying an {product-title} cluster. The bootstrap VM connects to the `baremetal` network and to the `provisioning` network, if present, via the network bridges.
+Installer-provisioned installation on bare metal nodes deploys and configures the infrastructure that a {product-title} cluster runs on. This guide provides a methodology to achieving a successful installer-provisioned bare-metal installation. The following diagram illustrates the installation environment in phase 1 of deployment.
```
Comment (stevsmit) [inline installing/installing_bare_metal_ipi/ipi-install-overview.adoc:7]: deployment:

**Comment by tmulquee** [comment]
I will examine what's going on. I've been working mainly with new content to date and I'm having some difficulty cherry-picking and targeting older releases.

File: installing/installing_ibm_cloud/install-ibm-cloud-prerequisites.adoc (hunk 2 above)
Comment (tmulquee) [reply installing/installing_ibm_cloud/install-ibm-cloud-prerequisites.adoc:16]: Done.

**Comment by tmulquee** [comment]
Closing this since the parent PR needed a round 2 of edits. The revision will be updated with a fresh round of cherry-picking.

**Pull request closed without merging**